crates/parcel/src/parcel.rs

```diff
@@ -3,22 +3,29 @@ use std::sync::Arc;
 use parcel_filesystem::os_file_system::OsFileSystem;
 use parcel_filesystem::FileSystemRef;
 use parcel_package_manager::NodePackageManager;
+use parcel_plugin_rpc::RpcHostRef;
 
 pub struct Parcel {
   pub fs: FileSystemRef,
+  pub rpc: Option<RpcHostRef>,
 }
 
 pub struct ParcelOptions {
   pub fs: Option<FileSystemRef>,
+  pub rpc: Option<RpcHostRef>,
```

Comment:
The difference with Mach is I used a `HashMap<String, RpcHostRef>` to allow mutliple plugin execution runtimes to run simultaniously which you could select from the `.parcelrc` 
```yaml
resolvers:
  "nodejs:./plugins/resolver.js" # nodejs is the default when "nodejs:" omitted
  "wasm:./plugins/resolver.wasm"
  "rlib./plugins/resolver.rlib"
```